canonical/ubuntu.com
# CVE search results clean-up

## Done

- Fixed remove filter bug
- Refactored stale JS
- Reinstated package query functionality
- Deleted stale templates
- Added no results view

## QA

- View the site locally in your web browser at: https://ubuntu-com-13893.demos.haus/security/cves?q=
- Test filtering functionality
- Test querying by affected package
- Test querying by invalid cve and see that the no results notification looks as expected
- See that all works as expected

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-11310, https://warthogs.atlassian.net/browse/WD-8041
